cmd/sysl/cmd_protobuf.go

```diff
@@ -20,9 +20,9 @@ func (p *protobuf) Name() string       { return "protobuf" }
 func (p *protobuf) MaxSyslModule() int { return 1 }
 
 func (p *protobuf) Configure(app *kingpin.Application) *kingpin.CmdClause {
-	cmd := app.Command(p.Name(), "Generate textpb/json").Alias("pb")
+	cmd := app.Command(p.Name(), "Generate textpb/json/binary").Alias("pb")
 	cmd.Flag("output", "output file name").Short('o').Default("-").StringVar(&p.output)
-	opts := []string{"textpb", "json"}
+	opts := []string{"textpb", "json", "pb"}
```

Comment:
Why `pb` and not `binary`, as suggested by the description?

Reply:
Follow the existing naming convention, for example type `textpb`.
Also refer https://github.com/arr-ai/arrai/issues/478#issuecomment-646589243